WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -302,6 +302,14 @@ class AppSettingsViewController: UITableViewController {
             ActionDispatcher.dispatch(NoticeAction.post(notice))
         }
     }
+
+    func presentWhatIsNew() -> ImmuTableAction {
+        return { [weak self] row in
+            let controller = WhatIsNewViewController()
```

Comment:
If you add a `guard let self = self else { return }` above this line, we can avoid instantiating the VC when self is `nil`.

Definitely not necessary for the code to behave though, so feel free to ignore if you prefer this way.

Reply:
Totally agree! Good catch!!